core_backend/app/question_answer/routers.py

```diff
@@ -253,6 +257,7 @@ async def search(
 @classify_safety__before
 @translate_question__before
 @paraphrase_question__before
+@generate_tts__after
 @generate_llm_response__after
 @check_align_score__after
```

Comment:
wait what was the conclusion here? Shouldn't it be:

```
@generate_tts__after
@check_align_score__after
@generate_llm_response__after
```

Reply:
Sid did tell me that the` after` decorators work in the reverse order, but as align score was not being accessed at all due to some issue which ig carlos figured it out today, i did not change it for now but yes the `align score` decorator should be placed above the` llm_response` one